### PR TITLE
Fix import format at pandas/tests/extension directory

### PR DESCRIPTION
**Pull request by mwoss**
- [x] partial #23334 
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

PR includes 21 files, but I think it's acceptable and unnecessary to divide it to two separate PRs.

### CODE REVIEW

File: pandas/tests/extension/arrow/bool.py [hunk 1]
```diff
@@ -9,10 +9,11 @@
 import itertools
 
 import numpy as np
```
Comment (jreback) [inline pandas/tests/extension/arrow/bool.py]: you can use

pytest.importorskip here

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/extension/arrow/bool.py (hunk 1 above)
Comment (mwoss) [reply pandas/tests/extension/arrow/bool.py]: pytest.importorskip have to be placed before exact import, but I don't think so it's possible now. Isort won't allow to do that.

**Comment by mwoss** [comment]
After adding pytest.importorskip again, build is still failing. (probably PEP8 code style issue)

**Comment by thoo** [comment]
`isort` thought these two are not properly sorted. 
```

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @mwoss

**Comment by osdnk** [comment]
@mwoss 
👏

**Pull request merged**